egs/ami/s5/run_sdm.sh

```diff
@@ -10,17 +10,16 @@ mic=sdm$micid
 stage=0
 . utils/parse_options.sh
 
-# Set bash to 'debug' mode, it will exit on :
-# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
-set -e
-set -u
-set -o pipefail
-set -x
+# Set bash to 'debug' mode, it prints the commands (option '-x') and exits on : 
+# -e 'error', -u 'undefined variable', -o pipefail 'error in pipeline',
+set -euxo pipefail
 
 # Path where AMI gets downloaded (or where locally available):
-[ ! -r conf/ami_dir ] && echo "Please, run 'run_prepare_shared.sh' first!" && exit 1
-AMI_DIR=$(cat conf/ami_dir)
+AMI_DIR=$PWD/wav_db # Default,
```

Comment:
It is convenient to have push_button scripts. I was suggesting something like this where you have a default AMI_DIR and then update if the domain is known.

<pre><code>
AMI_DIR=$PWD/wav_db # Default,
case "$(hostname -d)" in
fit.vutbr.cz) # BUT cluster,
AMI_DIR=/mnt/scratch05/iveselyk/KALDI_AMI_WAV
;;
clsp.jhu.edu) # JHU cluster
AMI_DIR=/export/corpora4/ami/amicorpus
;;
*) echo "Using defaults location for AMI_DIR,"
;;
esac

</code></pre>